docs/semgrep-cloud-platform/user-management.md

```diff
@@ -54,7 +54,18 @@ To create an organization account:
 
 1. Sign in to Semgrep Cloud Platform.
 2. On the sidebar, click **your account name > Add org**.
-3. Complete the steps to finish adding your organization account.
+3. Choose either Create an organization or Join an organization.
+4. For **Create an organization**, follow these steps:
+    1. Enter the **Organization Display Name**.
+    2. Click **Create**.
+    3. Click **your account name** > **the new organization** you just created.
+    3. Click **Settings** > **Source code**.
+    4. Click on your Source code manager, for example Connect to GitHub.
```

Comment:
Do we want to mark these as "optional"? If not, we may want to explain what the purpose of "Connecting an SCM" does: it allows your Semgrep organization's membership to be managed by GH/GL. i.e. after linking your org, other members of your GH Organization/GL Group should be able to join your org by following step 5 below.

Reply:
I see what you mean, I'll separate the procedure into 2 as they do 2 different things. Thanks!